_view/attachChildViews.md

```diff
@@ -3,4 +3,4 @@ header: attachChildViews
 example: View.attachChildViews()
 ---
 
-Removes all child views, then looks at all of the child views that are expected and attaches each view.
+Removes all child views, then looks at all of the expected child views and attaches each view.
```

Comment:
how about...

Removes all the child views, finds all instances of the views child views, and calls [attach](#attach) on them.

? i feel like the "looks at all of the expected child views" isn't so great (on my part). do you think the above makes it a bit easier? and less of "but how does it look for them?"-y